### PR TITLE
H.kiessler

### PR DESCRIPTION
**Pull request by hans0**

### CODE REVIEW

File: HeroArena-App/src/app/roster.service.ts [hunk 1]
```diff
@@ -54,14 +54,25 @@ export class RosterService {
...
 
+  getGladiators() : Array <Gladiator>{
+    return this.gladiators;
+  }
+
+/*
   getGladiators(){
     return this.gladiators;
   }
+<<<<<<< HEAD
```
Comment (LuisV) [inline HeroArena-App/src/app/roster.service.ts]: This should not be here

File: HeroArena-App/src/app/roster-display/roster-display.component.ts [hunk 2]
```diff
@@ -17,15 +19,37 @@ export class RosterDisplayComponent implements OnInit {
...
   ngOnInit() {
-    //this.rs.getGladiators();
     this.rs.getUserGladiators(this.as.getUser().id).subscribe((userObj: Object) => {
-      console.log(userObj);
+      // console.log(userObj);
       this.rs.setGladiators(userObj);
+
+      this.gladiators = userObj;
+
+      // console.log(userObj[0]);
```
Comment (LuisV) [inline HeroArena-App/src/app/roster-display/roster-display.component.ts]: Let's try not to leave code comments, or at least write a comment about the significance of the code comments.

File: HeroArena-App/src/app/roster-display/roster-display.component.html [hunk 3]
```diff
@@ -1,3 +1,21 @@
 <p>roster-display works!</p>
```
Comment (LuisV) [inline HeroArena-App/src/app/roster-display/roster-display.component.html]: Let's remove this.

File: HeroArena-App/src/app/roster.service.ts (hunk 1 above)
Comment (hans0) [reply HeroArena-App/src/app/roster.service.ts]: I know.

**Pull request merged**